src/core/linux/SDL_dbus.c

```diff
@@ -192,6 +195,7 @@ SDL_DBus_Quit(void)
 #endif
     SDL_zero(dbus);
     UnloadDBUSLibrary();
+    SDL_free(inhibit_handle);
```

Comment:
```suggestion
    SDL_free(inhibit_handle);
    inhibit_handle = NULL;
```

(Does SDL have a nice macro for this, like GLib's `g_clear_pointer()`?)

Reply:
Nope, apparently there isn't something similar in SDL.